lib/delete-builder.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { QueryConditionsBuilder } = require('./query-conditions-builder');
+const { SelectBuilder } = require('./select-builder');
+
+class DeleteBuilder extends QueryConditionsBuilder {
+  constructor(params, options) {
+    super(params, options);
+    this.operations = {
+      from: null,
+    };
+  }
+
+  from(tableName, alias) {
+    const table = this.escapeIdentifier(tableName);
+    this.operations.from = { table, alias: alias && this.escapeKey(alias) };
+    return this;
+  }
+
+  _whereValueMapper(subquery) {
+    return typeof subquery === 'function'
+      ? subquery(new SelectBuilder(this.params, this.options))
+      : subquery;
+  }
+
+  build() {
+    let query = 'DELETE';
+
+    const from = this.operations.from;
+    if (!from) {
+      throw new Error('Cannot generate SQL, tableName is not defined');
+    }
+    query += String(from.table) + (from.alias ? `AS ${from.alias}` : '');
```

Comment:
Looks like `from.table` will be string already
```suggestion
    const alias = from.alias ? `AS ${from.alias}` : '';
    query += `$from.table $alias`;
```

Reply:
Done.